scripts/mail-all-activity.py

```diff
@@ -1,181 +0,0 @@
-#!/usr/bin/env python
```

Comment:
1. Does this mean we have to run manage.py on the server that hosts the 
production database?
2. Where are the required env vars documented?